src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -240,4 +232,44 @@ public static void copyDirectory(Path source, Path destination) throws IOExcepti
         }
         return lines;
     }
+
+    /**
+     * Return the hashmap associated with this nameFile.
+     * If a file named <code>nameFile</code> exists, and it contains  a serialized version of a Map, this
+     * Map will be returned.
+     * If the file doesn't exist or is corrupt, this method returns null
+     * @param nameFile
+     * @return the Map contained in the file named nameFile, or null in all other cases.
+     */
+    static Map<String, String> getHashMap(String nameFile) {
+        Map<String, String> hashes = null;
+        if (!Files.exists(Paths.get(nameFile))) {
+            return null;
+        }
+        try (FileInputStream fis = new FileInputStream(new File(nameFile));
+             ObjectInputStream ois = new ObjectInputStream(fis)) {
+            hashes = (Map<String, String>) ois.readObject();
+        } catch (IOException | ClassNotFoundException e) {
+            Logger.logDebug("Exception trying to get hashmap for "+nameFile+": "+e);
+            return null;
+        }
+        return hashes;
+    }
+
+    static String calculateCheckSum(File file) {
```

Comment:
add some doc, what is this method doing?

Reply:
Yes. This might need to move to `FileOps`  too

Reply:
Done